docs/rules/key-spacing.md

```diff
@@ -183,6 +184,106 @@ call({
 });
 ```
 
+### align
+
+The `align` option can take additional configuration through the `beforeColon`, `afterColon`, `mode`, and `on` options.
+
+If `align` is defined as an object, but not all of the parameters are provided, undefined parameters will default to the following:
+
+```js
+// Defaults
+align: {
+  "beforeColon": false,
```

Comment:
For consistency, ESLint uses 4-space indents, but it looks like the code examples here are using 2 spaces.